com.ibm.jbatch.tck.exec/default.tck.exec.properties

```diff
@@ -29,27 +28,21 @@ com.ibm.jbatch.spi.ServiceRegistry.BATCH_THREADPOOL_SERVICE=com.ibm.jbatch.conta
 com.ibm.jbatch.spi.ServiceRegistry.CONTAINER_ARTIFACT_FACTORY_SERVICE=com.ibm.jbatch.container.services.impl.WeldSEBatchArtifactFactoryImpl
 #com.ibm.jbatch.spi.ServiceRegistry.DELEGATING_ARTIFACT_FACTORY_SERVICE=com.ibm.jbatch.container.services.impl.DelegatingBatchArtifactFactoryImpl
 com.ibm.jbatch.spi.ServiceRegistry.J2SE_MODE=true
-
 # Test Class: ExecutionTests
 ExecutionTests.testInvokeJobWithUserStop.sleep=1000
```

Comment:
You don't like the spacing in the properties files?    I thought it was a bit more readable but not too attached either way.